includes/minimal_header_template.php

```diff
@@ -0,0 +1,110 @@
+<?php
+// try{
+    // if($LANG_TAG == 'en' || !file_exists($SERVER_ROOT.'/content/lang/header.' . $LANG_TAG . '.php')) include_once($SERVER_ROOT . '/content/lang/header.en.php');
+    // else include_once($SERVER_ROOT . '/content/lang/header.' . $LANG_TAG . '.php');
+// } 
+// catch(Exception $error){
+//     // var_dump($error);
+// }
+include_once($SERVER_ROOT . '/includes/head.php');
+
+include_once($SERVER_ROOT . '/classes/ProfileManager.php');
+$pHandler = new ProfileManager();
+$isAccessiblePreferred = $pHandler->getAccessibilityPreference($SYMB_UID);
+$SHOULD_USE_HARVESTPARAMS = $SHOULD_USE_HARVESTPARAMS ?? false;
+$collectionSearchPage = $SHOULD_USE_HARVESTPARAMS ? '/collections/index.php' : '/collections/search/index.php';
+?>
+<div class="header-wrapper" style="position:absolute;top:10; width: 100vw;">
```

Comment:
Based on the name of the branch I am going to assume that the position is supposed to be sticky instead of absolute. Top all has no value so it isn't doing anything. If it was intended to be absolute just to be at the top of the page you should just make a regular relative block element that way you can avoid all of the custom spacing on every page it is implemented on, btw sticky will also take care of this.

Reply:
Good catch. Initially, we were thinking of making this sticky, but on the map pages, it doesn't really make sense since the viewport of the map doesn't change (i.e., users are just zooming and dragging with the map, rather than scrolling the page). I just never re-named the branch. I removed `position:absolute;top:10;` in  ebe2ae81c.